python/importer/main.py

```diff
@@ -6,27 +6,26 @@
 import csv
 import zstandard
 from datetime import datetime, timedelta
-import pytz
-from dateutil import parser
 import psycopg2 as psycopg
-from common.utils import get_conn_params
+from common.utils import get_conn_params, get_logger
 from .run_analysis import main as run_analysis
 
 # TODO: import other event types as well when needed.
 event_types_to_import = ['DOC', 'DOO']
 
 def main(dataImporter: func.TimerRequest):
+    logger = get_logger()
+    logger.debug("###   Going to run importer.   ###")
     imported_successfully = False
     conn = psycopg.connect(**get_conn_params())
     try:
         with conn:
             with conn.cursor() as pg_cursor:
                 imported_successfully = import_data_to_db(pg_cursor=pg_cursor)
-                print(f' imported_successfully {imported_successfully}')
     finally:
         conn.close()
         if imported_successfully == True:
-            print("Import done successfully, running analysis.")
+            print("###   Import done successfully.   ###")
```

Comment:
`log.debug("### Import done successfully. ###")` (or `log.info()`, might be good to log successful import runs in production so they're easily trackable)